docs/pages/components/accordion.mdx

```diff
@@ -1,10 +1,9 @@
-import SEO from "../components/SEO"
```

Comment:
`SEO` is still used, but we instead query render it in in `src/templates/docs.js` using the page's `frontmatter`.